src/codeowners.md

```diff
@@ -0,0 +1,199 @@
+  * NIO: [`nio-dev@openjdk.java.net`](mailto:nio-dev@openjdk.java.net)
+* HotSpot: [`hotspot-dev@openjdk.java.net`](mailto:hotspot-dev@openjdk.java.net)
+  * Compiler: [`hotspot-compiler-dev@openjdk.java.net`](mailto:hotspot-compiler-dev@openjdk.java.net)
+  * GC: [`hotspot-gc-dev@openjdk.java.net`](mailto:hotspot-gc-dev@openjdk.java.net)
+  * Runtime: [`hotspot-runtime-dev@openjdk.java.net`](mailto:hotspot-runtime-dev@openjdk.java.net)
+  * Serviceability: [`serviceability-dev@openjdk.java.net`](mailto:serviceability-dev@openjdk.java.net)
+* I18n: [`i18n-dev@openjdk.java.net`](mailto:i18n-dev@openjdk.java.net)
+* Langtools
+  * Amber Project: [`amber-dev@openjdk.java.net`](mailto:amber-dev@openjdk.java.net)
+  * Javac: [`compiler-dev@openjdk.java.net`](mailto:compiler-dev@openjdk.java.net)
+  * Javadoc: [`javadoc-dev@openjdk.java.net`](mailto:javadoc-dev@openjdk.java.net)
+  * JShell: [`kulla-dev@openjdk.java.net`](mailto:kulla-dev@openjdk.java.net)
+  * Panama Project: [`panama-dev@openjdk.java.net`](mailto:panama-dev@openjdk.java.net)
+  * Valhalla Project: [`valhalla-dev@openjdk.java.net`](mailto:valhalla-dev@openjdk.java.net)
+* Security: [`security-dev@openjdk.java.net`](mailto:security-dev@openjdk.java.net)
+
+## Directory to area mapping
+
+* `make` – Build team
+* `hotspot` – HotSpot
+  * `cpu` – Compiler, Runtime
+  * `jdk.*` – Compiler
+  * `os` – Runtime
+  * `os_cpu` – Compiler
+  * `share`
+    * `adlc` – Compiler
+    * `aot` – Compiler
+    * `asm` – Runtime
+    * `c1` – Compiler
+    * `ci` – Compiler
+    * `classfile` – Runtime
+    * `code` – Compiler
+    * `compiler` – Compiler
+    * `gc` – GC
+    * `include` – HotSpot
+    * `interpreter` – Runtime
+    * `jfr` – Runtime
+    * `jvmci` – Compiler
+    * `libadt` – Compiler
+    * `logging` – Runtime
+    * `memory` – Runtime, GC
+    * `metaprogramming` – Runtime
+    * `oops` – Runtime
+    * `opto` – Compiler
+    * `precompiled` – Runtime
+    * `prims` – Runtime, Serviceability
+    * `runtime` – Runtime
+    * `services` – Runtime
+    * `shark` – Compiler
+    * `trace` – Runtime
+    * `utilities` – Runtime
+* `java.base`
+  * Core Libs should almost always be included but Langtools, HotSpot, and/or Security may also be involved.
+  * `classes`
+    * `crypto` – Security
+    * `internal` – HotSpot, Core Libs
+    * `invoke` – Core Libs
+    * `io` – NIO
+    * `lang` – Core Libs
+    * `launcher` – Langtools
+    * `math` – Core Libs
+    * `net` – Net
+    * `nio` – NIO
+    * `reflect` – Core Libs
+    * `security` – Security
+    * `text` – Core Libs
+    * `time` – Core Libs
+    * `util` – I18n, Core Libs
+  * `conf`
+    * `sdp` – Net
+    * `security` – Security
+  * `legal` –
+  * `lib/security` – Security
+  * `man` –
+  * `native`
+    * `common` –
+    * `include` – Runtime, Core Libs
+    * `jspawnhelper` – LangTools
+    * `launcher` – Langtools
+    * `libfdlibm` –
+    * `libjava` – Core Libs
```

Comment:
Curious.  libjava contains code for the launcher.  In other parts of this list, launcher code is being sent to LangTools.  It's possible that this split is intentional...